src/Fabulous.Core/ViewConverters.fs

```diff
@@ -626,8 +626,8 @@ module Converters =
         | ValueSome prev, ValueNone -> updateFunc prev defaultValue
         | ValueNone, ValueSome curr -> updateFunc defaultValue curr
 
-    /// Update the TitleView of a NavigationPage, given previous and current values
-    let internal updateNavigationPageTitleView (prevOpt: ViewElement voption) (currOpt: ViewElement voption) (target: Xamarin.Forms.NavigationPage) =
+    /// Update the attached NavigationPage.TitleView property of a Page, given previous and current values
+    let internal updateNavigationPageTitleView (prevOpt: ViewElement voption) (currOpt: ViewElement voption) (target: Page) =
```

Comment:
I guess change the name to `updatePageTitleView`?

Reply:
I left it as it was because the TitleView property is on NavigationPage, but you're right, I'll change it.